rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -925,7 +929,7 @@ void game_list_frame::resizeEvent(QResizeEvent *event)
 }
 
 /**
- Cleans and readds entries to table widget in UI.
+ Cleans and reads entries to table widget in UI.
```

Comment:
:) this was no typo. But it's fine either way

Reply:
What typo? Shouldn't it stay as  `readds` as in `re-adds`?

Reply:
Oops, sorry about that. I reverted this change.